pkg/sessions/dynamodb/client.go

```diff
@@ -0,0 +1,74 @@
+package dynamodb
+
+import (
+	"context"
+	b64 "encoding/base64"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	dynamo "github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"time"
```

Comment:
This will get picked up by the linter, but imports from the standard library should be grouped together, alphabetically, then there should be a blank line, then extra imports (those starting with github.com) should follow that